index.html

```diff
@@ -459,6 +462,164 @@
+              resolved with <code>true</code> by <code><a for="RemotePlayback">connect</a>()</code> method.
+              The local playback of the media element continues in this
+              state and media commands still take effect on the local playback.
+            </li>
+            <li>
+              <dfn>connected</dfn> means that the transition from local to
+              <a>remote playback</a> has finished and all media commands now
+              take effect on the playback state of the <a>remote playback device</a>
+            </li>
+            <li>
+              <dfn>disconnected</dfn> means that the <a>remote playback</a>
+              has not been initiated, has failed to initiate or has been
+              stopped. All media commands will take effect on the local
+              playback in this state. The <a>remote playback</a> can be
+              initiated through a call to <code><a for="RemotePlayback">connect</a>()</code>.
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h4>
+            Establishing a connection with a remote playback device
+          </h4>
+          <p>
+            When the <a>user agent</a> is to <dfn>establish a connection with the remote playback device</dfn>,
+            it MUST run the following steps:
+          </p>
+          <dl>
+            <dt>
+              Input
+            </dt>
+            <dd>
+              <var>remotePlayback</var>, the <code>RemotePlayback</code> object that is to be connected.
+              The <a>remote playback state</a> of <var>remotePlayback</var> must be
+              <a for="RemotePlaybackState">connecting</a>.
+            </dd>
+          </dl>
+          <ol>
+            <li>
+              Request connection of <var>remotePlayback</var> to the <a>remote playback device</a>.
+            </li>
+            <li>
+              If connection completes successfully, <a>queue a task</a> to run the following steps:
+              <ol>
+                <li>
+                  Set the <a>remote playback state</a> of <var>remotePlayback</var> to
+                  <a for="RemotePlaybackState">connected</a>.
+                </li>
+                <li>
+                  <a>Fire a simple event</a> named <a>statechange</a> at <var>remotePlayback</var>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          <div class="note">
+            The mechanism that is used to connect the user agent with the <a>remote playback device</a> and
+            play the <a>remote playback source</a> is an implementation choice of the user agent. The
+            connection must provide a two-way messaging abstraction capable of carrying media commands to
+            the remote playback device and receiving media playback state in order to keep the remoted
+            media element state in sync.
+          </div>
+          <div class="note">
```

Comment:
I don't think "Notes" are normative. If we want to explicitly mention a behaviour, we might want it to be part of normative text.